cmd/venova/main.go

```diff
@@ -1,23 +1,35 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"os"
-	"venova/bot"
-	"venova/db"
+
+	"github.com/braydencw1/venova"
+	"github.com/braydencw1/venova/bot"
+	"github.com/braydencw1/venova/db"
 
 	"github.com/bwmarrin/discordgo"
 	"github.com/joho/godotenv"
 )
 
 func main() {
+	showVersion := flag.Bool("version", false, "Display version information")
```

Comment:
Grab a library for flags the builtin one is basically useless. I've been using [kong](github.com/alecthomas/kong) and I like it so far.